package.json

```diff
@@ -27,7 +27,8 @@
     "lint:fix": "eslint --fix -f ./node_modules/eslint-friendly-formatter \"{,**/}*.{js,vue}\" && stylelint \"**/*.{css,vue}\" --fix",
     "test": "yarn run lint && yarn test:unit && yarn test:e2e",
     "test:unit": "cross-env NODE_ENV=testing jest --maxWorkers=2",
-    "test:e2e": "cross-env yarn run pack && tape \"${TEST:-test/e2e/*.js}\"",
+    "test:e2e": "(kill $(lsof -t -i:9515) || true) && nightwatch --config ./test/e2e/nightwatch.json",
```

Comment:
ugly and not cross platform. but I couldn't fix it in nightwatch :/

Reply:
A nightwatch side project owner said he will fix it in a week (on january 2016)(https://github.com/mucsi96/nightwatch-cucumber/issues/12#issuecomment-174865979) let's keep this dirty stuff XD